multy_core/src/bitcoin/bitcoin_transaction.cpp

```diff
@@ -373,7 +373,12 @@ class BitcoinTransactionFee
                   properties,
                   "amount_per_byte",
                   Property::OPTIONAL,
-                  verify_non_negative_amount),
+                  [this](const BigInt& amount) {
+                      if (amount < BigInt("2"))
```

Comment:
1. While inside multy_core, you can construct `BigInt` from `int32` and\or `int64`: `BigInt(2)`
2. Please fix exception message, to something like: "Value should be >2"